sources/assets/Stride.Core.Assets.CompilerApp/build/Stride.Core.Assets.CompilerApp.targets

```diff
@@ -160,6 +160,11 @@
   </Target>
 
   <!-- Clean assets -->
+  <Target Name="StrideCleanAssetsOnClean" AfterTargets="Clean">
```

Comment:
nit: Prehabs simply "StrideCleanAssets" ?

Reply:
That was what I went with initially but there's already a target named `StrideCleanAsset`